### DISCUSSION TITLE
How to save the sample file

### DISCUSSION THREAD

**Discussion by Hellobamboobamboo** [lnccbrown/HSSM#542] [General]
It showed that the type is arviz.InferenceData object. I tried:
import arviz as az

# Assuming 'inference_data' is your arviz.InferenceData object
inference_data.to_netcdf("inference_data.nc")

print("InferenceData saved as 'inference_data.nc'")

to save the file, but it didnt work. Any ideas?

**Comment by digicosmos86** [comment]
Use the traces object from your .sample() call. For example, if you have run traces = model.sample(), then traces is your InferenceData object. Run traces.to_netcdf("the_file_name_you_want_to_save_to.nc"), then you'll have a file saved to the same directory
